### PR TITLE
Readme.md and new makefile.

### PR DESCRIPTION
**Pull request by abt2804**
- New Makefile (unfortunately no .rdef)
- New Readme.md with screenshot.;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Simply go into source directory and type ``make``.
+
+
+Copyright
+------------
+
+Original 2000 version by Jonathan Villemure, enhanced and updated for Haiku in 2018 by Arnav Bhatt.
+
+
+Contact
```
Comment (scottmc) [inline README.md]: Drop contact info.

Comment (abt2804) [reply README.md]: Done

Comment (scottmc) [reply README.md]: Enhanced and updated?  All you are doing if updating the readme.  This part can be dropped.  Your updates with be tagged with you as the author of the commit(s) you make.  No need to add it to readme unless you've made significant changes to the code.

Comment (abt2804) [reply README.md]: Oops, I forgot about this one, wait a minute.

**Comment by scottmc** [comment]
Just noticed that there is an rsrc file.  Please open a new pull request and convert that to rdef and update the makefile to use it instead.  See https://www.haiku-os.org/documents/dev/compile_them_resources/

**Comment by abt2804** [comment]
> Just noticed that there is an rsrc file. Please open a new pull request and convert that to rdef and update the makefile to use it instead. See https://www.haiku-os.org/documents/dev/compile_them_resources/

I converted that already but I can't able to make it work.

**Comment by scottmc** [comment]
in the Makefile, try changing these lines:
```
#	Specify the resource definition files to use. Full or relative paths can be
#	used.
RDEFS =

#	Specify the resource files to use. Full or relative paths can be used.
#	Both RDEFS and RSRCS can be utilized in the same Makefile.
RSRCS = animator.rsrc```

putting in RDEFS = animator.rdef instead of RSRC = animator.rsrc

**Pull request merged**